app/assets/stylesheets/splash.scss

```diff
@@ -6,9 +13,171 @@
   -webkit-background-size: cover;
   background-size: cover;
   height: 50vh;
-  position: relative;
+  overflow: hidden;
+  box-shadow: 0 0 5px rgba(0,0,0,1);
 }
 
 .bottom {
   height: 50vh;
+  color: $black;
+}
+
+// *************************** TITLE BOX **************************************
+
+.title-box {
+  top: 20%;
+  height: 50%;
+  text-align: center;
+  position: relative;
+  width: 100%;
+}
+
+.title {
+  color: $white;
+  font-family: 'Josefin Sans', sans-serif;
+  font-size: 10vmin;
+}
+
+.subtitle {
+  font-size: 7vmin;
+}
+
+.action {
+  margin-top: 5vh;
+  font-family: 'Josefin Sans', sans-serif;
+  font-size: 7vmin;
+  background: transparent;
+  border: 3px solid $white;
+}
+
+@media only screen and (min-width: 40.063em) {
+  .title {
+    font-size: 7vmin;
+  }
+  .subtitle {
+    font-size: 5vmin;
+  }
+  .action {
+    font-size: 4vmin;
+  }
+}
+
+ @media only screen and (min-width: 90.063em) {
+   .title {
+     font-size: 5vmin;
+   }
+   .subtitle {
+     font-size: 4vmin;
+   }
+   .action {
+     font-size: 3vmin;
+   }
+}
+
+// *************************** DEMO BOX **************************************
+
+
+.demo-row {
+  position: relative;
+  height: 100%;
+}
+
+
+// *************************** ARROWS **************************************
+
+
+.arrow {
+  position: absolute;
```

Comment:
Properties should be ordered border, border-radius, color, cursor, font-family, font-size, height, padding, position, text-align, top, width